### PR TITLE
Added example info to README.md

### PR DESCRIPTION
**Pull request by yomexzo**
@minasmart @tessalt @harismahmood89

### CODE REVIEW

**Comment by minasmart** [comment]
Please add this as an examples section to [CONTRIBUTING.md](https://github.com/Shopify/js-buy-sdk/blob/master/CONTRIBUTING.md).

Not in the readme 😄

**Comment by yomexzo** [comment]
Done @minasmart. 😄

**Comment by minasmart** [comment]
LGTM! @tessalt or @harismahmood89 could you please take a look?

File: examples/README.md [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
 ```bash
 npm install && npm run examples
 ```
 
-Once it's running, open your browser to `http://localhost:4200/examples/cart/` to view the example.
-The example works in Internet Explorer 9+, Safari 5.1+, iOS 6.1+,
+Once it's running, open your browser to `http://localhost:4200/examples/<example-name>/` to view the example.
+*eg: `/examples/cart` for the cart example*.
+
+The examples works in Internet Explorer 9+, Safari 5.1+, iOS 6.1+,
```
Comment (tessalt) [inline examples/README.md]: `The examples work`

Comment (yomexzo) [inline examples/README.md]: 😸 thanks @tessalt .. corrected

**Comment by yomexzo** [comment]
hey @harismahmood89. 👀  please ... 😄

**Comment by minasmart** [comment]
👍

**Pull request merged**